data/mods/Magiclysm/items/ethereal_items.json

```diff
@@ -757,5 +757,45 @@
     "relic_data": {
       "passive_effects": [ { "has": "WIELD", "condition": "ALWAYS", "values": [ { "value": "ITEM_DAMAGE_PURE", "add": 10 } ] } ]
     }
+  },
+  {
+    "id": "azoth_sword",
+    "type": "GENERIC",
+    "symbol": "/",
+    "color": "light_gray",
+    "looks_like": "sword_wood",
+    "name": { "str": "Azoth" },
+    "description": "Azoth.",
+    "material": [ "hardsteel" ],
+    "flags": [ "UNBREAKABLE_MELEE", "NONCONDUCTIVE", "NO_REPAIR", "NO_SALVAGE", "MAGIC_FOCUS", "LIGHT_10", "TRADER_AVOID" ],
+    "techniques": [ "WBLOCK_1", "WIDE", "SPIN" ],
+    "weight": "1 g",
+    "volume": "1 ml",
+    "longest_side": "100 cm",
+    "bashing": 15,
+    "cutting": 45,
+    "to_hit": 2
+  },
+  {
+    "id": "ancile_shield",
+    "type": "ARMOR",
+    "name": { "str": "Ancile" },
+    "category": "armor",
+    "description": "Ancile",
```

Comment:
I don't have any suggestions yet; how do you envision it?

Reply:
about what? its just shield item that i copypasted from ice shield spell, its mostly same except some stuff like material

Reply:
Huh. I will have to take a look at the ice shield spell.

Reply:
okay, i asked him, and he says that shield must be too op (with which i agree but shields really was op before creating gun, and still useful in police), and asked me to rebuild it to aura with not so high stats, but with full body protection.